pkg/server/controller/start_ekadashi.go

```diff
@@ -0,0 +1,69 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Lighty0410/ekadashi-server/pkg/ekadashi"
+	"github.com/Lighty0410/ekadashi-server/pkg/mongo"
+)
+
+// FillEkadashi is a goroutine that autofill ekadashi dates via getEkadashi().
+// If succeed it adds ekadashi date to the database.
+// It gonna logs errors only. Not to return themselves.
```

Comment:
will log errors. also it is clear that it will not return errors b/c there is no return value at all